js/src/components/contact-information/store-address-card.js

```diff
@@ -76,7 +65,9 @@ const StoreAddressCard = ( {
 
 	const handleRefreshClick = () => {
 		setSaving( true );
-		updateGoogleMCContactInformation().catch( () => setSaving( false ) );
+		updateGoogleMCContactInformation()
+			.then( () => setSaving( false ) )
+			.catch();
```

Comment:
@joemcgill If there's an error, we'll still see the spinner since `isSaving` will be `true`

Reply:
Updated this in [a9f5530](https://github.com/woocommerce/google-listings-and-ads/pull/2661/commits/a9f5530026d5ffc8431532117163db32f7edf34c) to use `.finally()`, which is semantically better. Realized that errors are already [handled in the action itself](https://github.com/woocommerce/google-listings-and-ads/blob/d78738a200681c02f84070edbbe1aece640e8938/js/src/data/actions.js#L605-L614).